src/main/kotlin/org/openmbee/flexo/mms/routes/gsp/ModelRead.kt

```diff
@@ -23,6 +23,60 @@ suspend fun GspLayer1Context<GspReadResponse>.readModel(refType: RefType) {
         }
     }
 
+    val authorizedIri = "<${MMS_URNS.SUBJECT.auth}:${transactionId}>"
```

Comment:
The added things here are from a borked merge from develop, can just take them out